src/content/en/updates/2019/06/image-support-for-async-clipboard.md

```diff
@@ -180,15 +236,54 @@ document.addEventListener('copy', async (e) => {
 });
 ```
 
+### Demo
+
+<iframe style="border: solid 1px;" width="100%" height="500"
+  src="https://jsfiddle.net/0794oysr/2/embedded/result,js,html,css"
+  allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"
+></iframe>
+
+### Security
+
+Opening up the Asynchronous Clipboard API for images comes with certain
+[risks](https://w3c.github.io/clipboard-apis/#security) that need to be carefully evaluated.
+One new challenge are so-called [image compression bombs](https://bomb.codes/bombs#images),
+that is, image files that appear to be innocent, but—once decompressed—turn out to be huge.
+The [specification](https://w3c.github.io/clipboard-apis/#image-transcode)
+mentions transcoding as a mitigation method:
+*“To prevent malicious image data from being placed on the clipboard, the image data may be
+transcoded to produce a safe version of the image.”*
+There is some ongoing discussion happening on the related
+[W3C Technical Architecture Group review](https://github.com/w3ctag/design-reviews/issues/350)
+on whether and how the transcoding should be specified.
+
+## Next Steps
+
+Eventually, the Clipboard API shall be opened up for any kind of data.
```

Comment:
This sounds too definite. Perhaps something like: "We are working on expanding the Clipboard API so that it can support a larger number of data types."

Reply:
Thank you, weakened my statement and mostly used your wording.